marine.c

```diff
@@ -690,6 +723,10 @@ WS_DLL_PUBLIC int marine_add_filter(char *bpf, char *dfilter, char **fields, int
     return size;
 }
 
+WS_DLL_PUBLIC void public_free(void *ptr) {
```

Comment:
I know you need that for marine-python, but still :(
Is there any way to avoid that?

Reply:
There is only one other possible solution that I could find, which is to use `ctypes` to also load libc (or a different matching dll on other systems) and call `free` from there.